RecoEgamma/ElectronIdentification/src/ElectronDNNEstimator.cc

```diff
@@ -0,0 +1,275 @@
+#include "RecoEgamma/ElectronIdentification/interface/ElectronDNNEstimator.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <iostream>
+#include <fstream>
+#include <memory>
+
+ElectronDNNEstimator::ElectronDNNEstimator() : cfg_{} {}
+
+ElectronDNNEstimator::ElectronDNNEstimator(std::vector<std::string>& models_files,
+                                           std::vector<std::string>& scalers_files,
+                                           std::string inputTensorName,
+                                           std::string outputTensorName)
+    : cfg_{.inputTensorName = inputTensorName,
+           .outputTensorName = outputTensorName,
+           .models_files = models_files,
+           .scalers_files = scalers_files,
+           .log_level = "2"} {
+  nModels_ = cfg_.models_files.size();
+  debug_ = cfg_.log_level == "0";
+  initTensorFlowGraphs();
+  initScalerFiles();
+  LogDebug("EleDNNPFid") << "Ele PFID DNN evaluation with " << nModels_ << " models and " << nInputs_[0]
+                         << " variables --> LOADED";
+}
+
+ElectronDNNEstimator::ElectronDNNEstimator(const Configuration& cfg) : cfg_(cfg) {
+  // Init tensorflow sessions
+  nModels_ = cfg_.models_files.size();
+  debug_ = cfg_.log_level == "0";
+  initTensorFlowGraphs();
+  initScalerFiles();
+  LogDebug("EleDNNPFid") << "Ele PFID DNN evaluation with " << nModels_ << " models and " << nInputs_[0]
+                         << " variables --> LOADED";
+}
+
+void ElectronDNNEstimator::initTensorFlowGraphs() {
+  // configure logging to show warnings (see table below)
+  tensorflow::setLogging(cfg_.log_level);
+  // load the graph definition
+  LogDebug("EleDNNPFid") << "Loading " << nModels_ << " graphs";
+  for (const auto& model_file : cfg_.models_files) {
```

Comment:
I think that these need to be  FileInPath instead of strings; apparently the tests are not running from /src directory.